week-6/Homework/mandatory/2-quotegenerator/quotes.js

```diff
@@ -489,3 +501,40 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+
+// function getRandomQuote(callBack) {
+//   let i = (callBack(quotes));
+//   quote.textContent = quotes[i].quote;
+//   author.textContent = quotes[i].author;
+// }
+
+// button.addEventListener("click", getRandomQuote(x));
+
+// function getRandomQuote() {
+//   let i = [Math.floor(Math.random() * quotes.length)]
+  
+//   quote.textContent = quotes[i].quote;
+//   author.textContent = quotes[i].author;
+// }
+
+let quote = document.getElementById("quote");
+let author = document.getElementById("author");
+let button = document.getElementById("button");
+
+function pickFromArray(quotes) {
+  return [Math.floor(Math.random() * quotes.length)]
+}
+
+function getRandomQuote() {
+  let i = (pickFromArray(quotes));
+  quote.textContent = `"${quotes[i].quote}"`;
+  author.textContent = `-${quotes[i].author;
+}
+
+button.addEventListener("click", getRandomQuote);
+
+getRandomQuote();
+
+
+
```

Comment:
Your button is not generating a random Quote from the array. The issue is you haven't pas the function which triggers the random Quote. Needs to be fixed Leatitia.